packages/client/src/views/GrantDetails.vue

```diff
@@ -402,4 +402,7 @@ export default {
   white-space: nowrap;
   width: 1%;
 }
+.grants-details-table-fit-content:after {
```

Comment:
Whoops, my bad, this also falls into the category of "things I tweaked from the design intentionally but forgot to call out in my review." 

I would argue the colons are extraneous and thus slightly hinder reading of the table. The point of the tabular layout is to make clear that the item in the left column describes the column in the right. That said, I don't feel strongly, so feel free to add the colons in. 

One very minor nit: If we decide we want the colons, they should probably be part of the text proper, rather than added in CSS. I haven't tested this, but I wouldn't be surprised if accessibility devices treat the CSS content differently than "real" content in the html of the page.

Reply:
I think you're right on that - which further supports your argument that they're extraneous :) I'll remove the colons for now!